core-plugins/src/test/java/io/cdap/plugin/batch/ETLTPFSTestRun.java

```diff
@@ -235,25 +233,6 @@ public void testOrc() throws Exception {
     // run the pipeline
     runETLOnce(appManager);
 
-    Connection connection = getQueryClient();
-    ResultSet results = connection.prepareStatement("select * from dataset_outputOrc").executeQuery();
-    results.next();
-
-    Assert.assertEquals("a", results.getString(1));
-    Assert.assertEquals(3.6f, results.getFloat(2), 0.1);
-    Assert.assertEquals(4.2, results.getDouble(3), 0.1);
-    Assert.assertEquals(true, results.getBoolean(4));
-    Assert.assertEquals(23456789, results.getLong(5));
-    Assert.assertArrayEquals(Bytes.toBytes("abcd"), results.getBytes(6));
-    Assert.assertEquals(12, results.getLong(7));
-    Assert.assertEquals("testUnion", results.getString(8));
-    Assert.assertNull(results.getString(9));
-    Assert.assertEquals(12, results.getLong(10));
-    Assert.assertEquals(0, results.getLong(11));
-    Assert.assertEquals(3.6f, results.getFloat(12), 0.1);
-    Assert.assertEquals(0.0, results.getFloat(13), 0.1);
-    Assert.assertEquals(4.2, results.getDouble(14), 0.1);
-    Assert.assertEquals(0, results.getDouble(15), 0.1);
```

Comment:
What is the purpose of `testOrc` test method, when all the assertions have been removed?
See if it is ok to delete the test?

Reply:
Typically a UT has 3 A's: Arrange, Act & Assert.
The code for assertion is removed which devalues the whole test. If the test was specific to explore or hive then I feel it can be removed. Please check this.
Or probably some new assertions can be added?

Reply:
This test doesn't seem to be related to explore service or hive queries. Were the assertions failing?